test/ci_workflow_gen.go

```diff
@@ -41,6 +41,7 @@ const (
 )
 
 var (
+	// Clusters 10, 25 and 26 are executed by the docker_test_cluster 10, 25, 26 workflows.
```

Comment:
should change 26 -> 27 in the list so that we don't have two clusters called 26 - docker_test_cluster_26 and `Cluster 26`